bin/scripts/lib/fonts.json

```diff
@@ -615,6 +615,17 @@
       "caskName": "victor-mono",
       "repoRelease": true,
       "description": "Clean, crisp and narrow, with a large x-height and clear punctuation"
+    },
+    {
+      "unpatchedName": "Geist Mono",
+      "RFN": false,
+      "patchedName": "GeistMono",
+      "folderName": "GeistMono",
+      "imagePreviewFont": "GeistMono Nerd Font",
+      "linkPreviewFont": "geist",
```

Comment:
This seems to be non-existing.
Is there a PR pending for that over there?

(The preview will link to `https://www.programmingfonts.org/#${linkPreviewFont}`)